examples/bakery_examples/pre_compile_compatibility/baked_add_compile.py

```diff
@@ -0,0 +1,45 @@
+from qualang_tools import baking
+from qm.qua import *
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from config import config
+
+# Create a first baked waveform, overridable
+# and which will be inserted in the config
+with baking(
+    config, padding_method="right", override=True, update_config=True
+) as b_template:
+    samples_I = [0.1, 0.1, 0.2, 0.1, 0.2]
```

Comment:
@arthurostrauss what sets here the fixed waveform duration? Is it coming from the config waveform duration? If so, what happens if the waveform duration which is given to `add_Op` is smaller/larger than the one in the config?

Reply:
The fixed waveform duration is indicated implicitly by the baking index provided as input. This baking index tells us which baked pulse to look at in the config in order to retrieve its length and assign it as a constraint length to the newly baked waveform.
If the new sample created is smaller than this constrained length, then 0s are padded (according to the padding method chosen) until the total duration matches the length constraint. If the newly baked samples are actually longer than the length constraint, an error is raised as you cannot use this waveform for an override based on the indicated waveform we wanted to modify. This indicated waveform is identified by the provided baking index as mentioned previously.